.tridactylrc

```diff
@@ -122,23 +122,23 @@
 " " details, read the comment at the top of this file.
 " fixamo_quiet
 " 
-" " Equivalent to `set csp clobber` before it was removed. This weakens your
-" " defences against cross-site-scripting attacks and other types of
-" " code-injection by reducing the strictness of Content Security Policy on
-" " every site in a couple of ways.
+" " Equivalent to `set csp clobber` before it was removed.
+" " This weakens your defences against cross-site-scripting attacks
+" " and other types of code-injection by reducing the strictness 
+" " of Content Security Policy onevery site in a couple of ways.
```

Comment:
I'm not sure what you wanted to change here but onevery isn't a word :)

Maybe "on all sites" would be clearer?

Reply:
thanks, fixed :) does this paragraph and the following ones relate to the line below or `fixamo_quiet`?
This is not quite clear to me here.

Reply:
It relates to the line below - the docs are directly above the commands

Reply:
Alright, added a clarification